media-gfx/graphviz/graphviz-2.50.0.ebuild

```diff
@@ -0,0 +1,274 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit autotools flag-o-matic java-pkg-opt-2 python-single-r1 qmake-utils
+
+DESCRIPTION="Open Source Graph Visualization Software"
+HOMEPAGE="https://www.graphviz.org/ https://gitlab.com/graphviz/graphviz/"
+# unfortunately upstream uses an "artifact" store for the pre-generated
+# tarball now, which makes predictable URLs impossible.
+SRC_URI="https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/${PV}/${P}.tar.xz"
```

Comment:
@SoapGentoo we should really ask if upstream would mind doing normal releases

Reply:
https://gitlab.com/graphviz/graphviz/-/merge_requests/2520#note_881272686